French translations

iNSiGhT59 opened the pull request:
I merged my two previous translations in this one (adding the forgotten backslash for ' )

Translations :
from line 4400 to 4627
from line 4636 to 4726
from line 4740 to 4746
from line 4754 to 4787
(line numbers from english string.xml)

iNSiGhT59 committed:
French translations

I merged my two previous translations in this one (with the forgotten backslash with ' )

Translations :
from line 4400 to 4627
from line 4636 to 4726
from line 4740 to 4746
from line 4754 to 4787
(line numbers from english string.xml)

gingo21 commented:
I think that translation have to be made in  res/values-fr/strings.xml

SferaDev commented:
Yes, it's not well formed! Please the strings file must be located on values-XX. In french on values-fr/strings.xml!

Also you have apostrophes without a backslash on them!

iNSiGhT59 commented:
Sorry...
All my attention was to not forget any backslash...and I finally edit the
wrong string.xml...
Shame on me...

I correct this as soon as I'm back home (tomorrow evening).
Le 15 juin 2013 16:41, "SferaDev" notifications@github.com a écrit :

> Yes, it's not well formed! Please the strings file must be located on
> values-XX. In french on values-fr/strings.xml!
> 
> —
> Reply to this email directly or view it on GitHubhttps://github.com/ParanoidAndroid/android_packages_apps_Settings/pull/26#issuecomment-19497461
> .

iNSiGhT59 committed:
French translations



I merged my two previous translations in this one (adding the forgotten backslash for ' )

Translations :
from line 4400 to 4627
from line 4636 to 4726
from line 4740 to 4746
from line 4754 to 4787
(line numbers from english string.xml)
